FashionConnect_api/routes/users.js

```diff
@@ -89,7 +90,8 @@ router.post('/users/profile', async (req, res) => {
       await user.save()
   
       // Return the user data in the response
-      res.json({ message: "User profile uodated succesfully" });
+      res.json({ user:user, message: "User profile uodated succesfully" });
+      console.log(res.json({user}))
```

Comment:
Let's make sure to remove the console.log when you put up PRs.

Reply:
Yes  i will henceforth.